README.md

```diff
@@ -182,6 +185,14 @@ The following parameters are available for the hiera class:
   Arbitrary YAML content to append to the end of the hiera.yaml config file.  
   This is useful for configuring backend-specific parameters.  
   Default: `''`
+* `provider`  
+  Which provider to use to install hiera-eyaml. Can be `puppetserver_gem` (PE 2015.x), `pe_puppetserver_gem` (PE 3.7 and 3.8), `pe_gem` (PE pre-3.7), `puppet_gem` (FOSS using puppet's gem), or `gem` (FOSS using system's gem)
```

Comment:
Isn't the puppetserver_gem compatible with both FOSS and PE systems that have Puppet >=4.0.0 ( Running Puppet Server )